log/logClient.go

```diff
@@ -206,7 +207,7 @@ func (fd *Feeder) WatchMessages(msgPath string, jsonFormat bool) error {
 		}
 	}
 
-	fmt.Fprintln(os.Stderr, "Stopped WatchMessages")
+	log.Fatalf("Stopped WatchMessages")
```

Comment:
We should not use log.Fatal. Let's return the error.

These functions are used in other tooling too like our ginkgo tests, it may result in unwanted behaviour in any external tooling. It should be the responsibility of whoever is using the function to handle the errors.